CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Ewon Ignition 7.9 Connector Changelog
 
+## v1.1.2
+### Major Changes
+* Added Flexy realtime read support
+
+### Minor Changes
+* Fixed an issue that caused the connector to fault when initially installed and no credentials configured
+* Fixed a security issue by removing user credentials from Ignition log files
+* Fixed an issue that caused issues with writing boolean tag values
```

Comment:
Fixed an issue that caused issues could be reworded.

Reply:
Don't know how I let this one slip by, but...fixed